src/app/teams/teams-view.component.html

```diff
@@ -86,8 +86,11 @@ <h3 class="margin-lr-3 ellipsis-title">{{team?.name}}</h3>
         <p i18n><b>Courses: ({{team?.courses?.length || 0}})</b></p>
         <p *ngIf="team?.courses?.length < 1; else courseList" i18n>There are no courses associated with this team.</p>
         <ng-template #courseList>
-          <mat-card *ngFor="let course of team?.courses">
+          <mat-card *ngFor="let course of team?.courses; let i = index">
             <a [routerLink]="'/courses/view/' + course._id">{{course.courseTitle}}</a>
+            <mat-card-actions>
+              <button mat-raised-button color="warn" (click)="team?.courses.splice(i, 1)" i18n>Remove</button>
```

Comment:
Good start adding this to the template, but this will not persist because it doesn't change the database.  You should update the document on the database to finish this PR

Reply:
Thank you for the feedback. I will continue working on it.

Reply:
So the function would look something like:

```
team.courses.splice(i, 1)
this.couchService.updateDocument('teams'/this.dbName, team).subscribe(() => { message })
```